### ISSUE TITLE
Processing aborted due to error 300010:1087106694; incident 7314215.

### ISSUE THREAD

**Issue by VantageKR** [snowflakedb/spark-snowflake#137]
When using the Snowflake Spark Connector, when trying to read the INFORMATION_SCHEMA views, I get this error:level 1 SQL Error [603] [XX000]: SQL execution internal error:
Processing aborted due to error 300010:1087106694; incident 7314215.

The query is reading from INFORMATION_SCHEMA.TABLES view, and doing a COPY INTO a previously created internal stage. The query is generated by the Spark Connector, latest version.
Here is some output produced by the failing Spark code:
19/08/22 20:04:57 ERROR SnowflakeStrategy: Pushdown failed :SQL execution internal error:
Processing aborted due to error 300010:1087106694; incident 7938892.
19/08/22 20:05:30 ERROR SnowflakeStrategy: Pushdown failed :SQL execution internal error:
Processing aborted due to error 300010:1087106694; incident 2356683.
net.snowflake.client.jdbc.SnowflakeSQLException: SQL execution internal error:
Processing aborted due to error 300010:1087106694; incident 9371821.
at net.snowflake.client.jdbc.SnowflakeUtil.checkErrorAndThrowExceptionSub(SnowflakeUtil.java:139)
at net.snowflake.client.jdbc.SnowflakeUtil.checkErrorAndThrowException(SnowflakeUtil.java:64)
at net.snowflake.client.core.StmtUtil.pollForOutput(StmtUtil.java:485)
at net.snowflake.client.core.StmtUtil.execute(StmtUtil.java:362)
at net.snowflake.client.core.SFStatement.executeHelper(SFStatement.java:504)
at net.snowflake.client.core.SFStatement.executeQueryInternal(SFStatement.java:249)
at net.snowflake.client.core.SFStatement.executeQuery(SFStatement.java:187)
at net.snowflake.client.core.SFStatement.execute(SFStatement.java:793)
at net.snowflake.client.jdbc.SnowflakeStatementV1.executeQueryInternal(SnowflakeStatementV1.java:258)
at net.snowflake.client.jdbc.SnowflakePreparedStatementV1.executeQuery(SnowflakePreparedStatementV1.java:166)
at net.snowflake.spark.snowflake.JDBCWrapper$$anonfun$executePreparedQueryInterruptibly$1.apply(SnowflakeJDBCWrapper.scala:248)
at net.snowflake.spark.snowflake.JDBCWrapper$$anonfun$executePreparedQueryInterruptibly$1.apply(SnowflakeJDBCWrapper.scala:246)
at net.snowflake.spark.snowflake.JDBCWrapper$$anonfun$1.apply(SnowflakeJDBCWrapper.scala:283)
at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24)
at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)

Note that I have already tried to deactivate query push down, with no effect.

Note also that I've tested on the latest version 1.5.2 with same results. The credentials have proper permission to create stages, the create stage succeeds, but the copy into fails.
Thanks in advance.

**Comment by VantageKR** [comment]
Version I'm testing with is 2.5.2, and the query on Snowflake side is:

```sql
-- succeeds
create  temporary stage if not exists identifier('spark_connector_unload_stage_3w8IioNXiK');

-- fails 
COPY INTO '@spark_connector_unload_stage_3w8IioNXiK/CXZ0z5qjU4/' FROM ( SELECT "ROW_COUNT", "CLUSTERING_KEY", "USER_DEFINED_TYPE_SCHEMA", "AUTO_CLUSTERING_ON", "LAST_ALTERED", "TABLE_TYPE", "TABLE_NAME", "TABLE_SCHEMA", "COMMENT", "COMMIT_ACTION", "CREATED", "USER_DEFINED_TYPE_NAME", "IS_TRANSIENT", "IS_INSERTABLE_INTO", "SELF_REFERENCING_COLUMN_NAME", "TABLE_OWNER", "IS_TYPED", "BYTES", "RETENTION_TIME", "REFERENCE_GENERATION", "TABLE_CATALOG", "USER_DEFINED_TYPE_CATALOG" FROM (SELECT * FROM INFORMATION_SCHEMA.TABLES) ) 
FILE_FORMAT = (
    TYPE=CSV
    COMPRESSION='gzip'
    FIELD_DELIMITER='|'
    FIELD_OPTIONALLY_ENCLOSED_BY='"'
    ESCAPE_UNENCLOSED_FIELD = none
    NULL_IF= ()
  )
   MAX_FILE_SIZE =  10000000; 
```

**Comment by Mingli-Rui** [comment]
This issue has been fixed from Spark Connector 2.6.0
